components/core/tools/scripts/lib_install/ubuntu-focal/install-prebuilt-packages.sh

```diff
@@ -12,11 +12,13 @@ DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
   checkinstall \
   cmake \
   curl \
+  diffutils \
```

Comment:
I see `diff` is already installed by default in Ubuntu containers, so do we need this?

Reply:
That's fine by me. I had added it while debugging, just to be sure, but we can drop it now. Is it default in centos as well or should I leave that one?

Reply:
`diff` used to be included in the centos container but it's not anymore. You can run the container with: `docker run --rm -it dokken/centos-stream-9 /bin/bash` and then check.

Reply:
Yeah, not in centos. Did you check both versions of ubuntu? Should I take it out of both of them?